02_knight_and_princess/roman_kuprii/main.cpp

```diff
@@ -8,35 +8,37 @@ using std::to_string;
 using std::vector;
 
 using EdgeId = int;
+using VertexId = int;
+
+constexpr int INVALID_ID = -1;
+const std::string JSON_GRAPH_FILENAME = "graph.json";
 
 struct Edge {
   EdgeId id;
```

Comment:
Давай добавим какое-нибудь дефолтное значение, чтобы у нас не оказалось мусора в памяти, если мы забудем инициализировать структуру
```suggestion
  EdgeId id = INVALID_ID;
```